documentation/aria.md

```diff
@@ -151,3 +151,16 @@
 ## `role="table"`
 
 - _Table_ component
+
+## `aria-description`
+
+`aria-description` wordt overgeslagen door veel hulpmiddelen om automatisch vertalingen van webpagina's te maken.
```

Comment:
`aria-description` gaat pas bestaan in [ARIA 1.3](https://w3c.github.io/aria/) (in Editor's Draft status), zou aannemen dat dat zeker nog een jaar duurt, misschien dus hier weglaten?